model/modelpb/apmevent.pb.json.go

```diff
@@ -65,6 +65,7 @@ func (e *APMEvent) MarshalFastJSON(w *fastjson.Writer) error {
 	}
 
 	if e.DataStream != nil {
+		doc.DataStream = &modeljson.DataStream{}
```

Comment:
I think if we declare the zero value struct above the `if` block, and only take a pointer to it in this block, then we can avoid an allocation. i.e.

```go
var dataStream modeljson.DataStream
if e.DataStream != nil {
    doc.DataStream = &dataStream
    ...
}
```